WMFData/Tests/WMFDataTests/WMFYearInReviewDataControllerTests.swift

```diff
@@ -381,6 +384,7 @@ final class WMFYearInReviewDataControllerTests: XCTestCase {
             let shouldShowEntryPoint = yearInReviewDataController.shouldShowYearInReviewEntryPoint(countryCode: frCountryCode, primaryAppLanguageProject: frProject)
 
             XCTAssertTrue(shouldShowEntryPoint, "Should show entry point when one personalized slide is enabled.")
+            // XCTAssertFalse(shouldShowEntryPoint) // Entry points disabled
```

Comment:
Tests should work independently of the implementation so we can test several situations.
Create different tests mocking both situations. That should be executed correctly regardless if the feature is active or not.